openscenario/openscenario_interpreter/include/openscenario_interpreter/simulator_core.hpp

```diff
@@ -380,26 +387,6 @@ class SimulatorCore
   class NonStandardOperation : private CoordinateSystemConversion
   {
   protected:
-    template <typename Performance, typename Properties>
-    static auto activatePerformanceAssertion(
-      const std::string & entity_ref, const Performance & performance,
-      const Properties & properties)
-    {
-      core->addMetric<metrics::OutOfRangeMetric>(entity_ref + "-out-of-range", [&]() {
-        metrics::OutOfRangeMetric::Config configuration;
-        configuration.target_entity = entity_ref;
-        configuration.min_velocity = -performance.max_speed;
-        configuration.max_velocity = +performance.max_speed;
-        configuration.min_acceleration = -performance.max_deceleration;
-        configuration.max_acceleration = +performance.max_acceleration;
-        configuration.min_jerk = properties.template get<Double>("minJerk", Double::lowest());
-        configuration.max_jerk = properties.template get<Double>("maxJerk", Double::max());
-        configuration.jerk_topic =
-          "/planning/scenario_planning/motion_velocity_optimizer/closest_jerk";
-        return configuration;
-      }());
-    }
```

Comment:
Why do you completely remove this section?

Reply:
Yes, this change does not affect the OpenSCENARIO syntax, so inside the interpreter, the part that used to call the api::addMetric function is simply changed to a function such as api::checkOutOfRange.
The change to the openscenario_interpretor package should be much smaller.

Reply:
It is advisable to REVERT this part and change the member functions of the API class that are called inside the activatePerformanceAssertion function.

Reply:
This change is related to a [comment here](https://github.com/tier4/scenario_simulator_v2/pull/979#issuecomment-1482449490) - please read it first.
As I understand it, you suggest adding the job to the `activatePerformanceAssertion(...)` - instead of [in the constructor](https://github.com/tier4/scenario_simulator_v2/blob/cf575c0fb9e9bbb6f62453fb4aa6fdf552b0eb4a/simulation/traffic_simulator/src/entity/vehicle_entity.cpp#L41). Do you think it is necessary?

Reply:
Yes, I have an activatePerformanceAssertion(...) function and do not see the need to change it